Snakefile

```diff
@@ -5,7 +5,7 @@
 ## 'anatomy', 'compound', 'disease', 'gene', 'protein'
 
 
-TERM_TYPES = ['anatomy', 'compound', 'disease', 'gene', 'protein']
+TERM_TYPES = ['gene']
```

Comment:
should this be kept?

Reply:
or was this just for testing?

Reply:
just for testing... usually i remember to not commit this :)